.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,3 +29,6 @@
 - [ ] I have added tests to cover my changes.
 - [ ] All new and existing tests passed.
 - [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).
+
+## Screenshots
```

Comment:
This feels like it really only belongs in Automate. Automate should be overriding the file to add this if necessary.

Reply:
True, would it make a difference if we said "Screenshots or Examples" ? Still will not apply to every PR, of course, but some. OK if you still want to delete, though.

Reply:
Oh, by the way, I do illustrations for a lot more than the UI. In this PR (https://github.com/chef/automate/pull/1875) I have "before" and "after" illustrations--for a database change!

Reply:
Oh, by the way, I do illustrations for a lot more than the UI. In this PR (https://github.com/chef/automate/pull/1875) I have "before" and "after" illustrations--for a database change!